### DISCUSSION TITLE
Option to change video quality

### DISCUSSION THREAD

**Discussion by Hooooooman** [pear-devs/pear-desktop#411] [Ideas]
I think there should be a setting in the three dots to increase or decrease video quality, cause I stats I saw the video quality is majorly decreased, I understand if there is a conflict.

**Comment by treasury** [comment]
I always want at least 1080p quality, the default video quality is very poor.

**Comment by Araxeus** [comment]
@Hooooooman @treasury 
Added a plugin  to allow changing quality (#446)

(You could close this discussion 🙂)

**Reply by Hooooooman** [reply]
can you guide me how to add a plugin by myself, because I'm not able to see the feature in v1.13.
![unknown](https://user-images.githubusercontent.com/91048741/138603517-6d902c67-4bee-4186-9f55-e676333257d2.png)
![unknown1](https://user-images.githubusercontent.com/91048741/138603535-f0648f01-9cf4-447e-9ea4-f0a519a0adcb.png)
0

**Reply by Hooooooman** [reply]
I'm sorry if I'm annoying or disturbing you in any way

**Reply by Araxeus** [reply]
Sorry I just added this feature last night, it will only be available in v1.14

New version coming soon with **many** new improvements and features 😁

**Reply by Hooooooman** [reply]
Can you give a approximate date when the v1.14 will be releasing??

**Reply by Araxeus** [reply]
> Can you give a approximate date when the v1.14 will be releasing?? 

You'll have to ask @th-ch (he is the project owner - im just a contributor)

**Reply by Araxeus** [reply]
@Hooooooman @treasury meanwhile if really you want better quality now (before the upcoming release)
you can enter devtools (`ctrl+shift+i` or `Options-Advanced-Toggle Developer Tools`) and paste the following in the console:
```js
var api = $('#movie_player');
api.setPlaybackQualityRange(api.getAvailableQualityLevels()[0]);
api.setPlaybackQuality(api.getAvailableQualityLevels()[0]);
```

this well set your video quality to the highest available 

youtube usually remembers your choice, so if you want always the highest - just play a 4k video and paste this code in console. then it will always try to reach 4k, defaulting to the highest available

**Reply by Araxeus** [reply]
New version has been released 🙂

**Comment by treasury** [comment]
Great! Thank you @Araxeus ! :)

**Comment by treasury** [comment]
@Araxeus Could you please create a plugin for this too: https://github.com/th-ch/youtube-music/discussions/460

It would be very nice.

**Reply by Araxeus** [reply]
Ill look into it :)

**Comment by inson1** [comment]
@treasury may you close it? :)
